soc/bouffalolab/common/soc_common.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021-2024 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file interrupt management code for riscv SOCs supporting the SiFive clic
+ */
+
+#ifndef __SOC_COMMON_H_
+#define __SOC_COMMON_H_
+
+/* clang-format off */
+
+/* IRQ numbers */
+#define RISCV_MACHINE_SOFT_IRQ       3  /* Machine Software Interrupt */
```

Comment:
This is introducing back some previous name for the macro, see `include/zephyr/arch/riscv/irq.h`